### ISSUE TITLE
Configuration of main js file in package.json

### ISSUE THREAD

**Issue by sharpner** [timseverien/taggd#53]
Hi, great project!

I am using taggd with webpack and currently stumble across this slight annoyance
```
WARNING in ../~/taggd/dist/taggd.js
Critical dependencies:
1:478-485 This seems to be a pre-built javascript file. Though this is possible, it's not recommended. Try to require the original source to get better results.
 @ ../~/taggd/dist/taggd.js 1:478-485
```
by importing the package as recommended in #52.

I think the configuration should point to the not-compiled js code. Would you accept a PR to change that behaviour or is there a reason for using the dist file in the configuration?

**Comment by timseverien** [comment]
The reason I chose to point to the compiled JS, is so workflows that doesn't transpile ES2015 can still import Taggd using `require('taggd')`. Changing the path of `main` would break that, if I'm not mistaken.

Shouldn't Webpack respect the [`module` field](https://github.com/timseverien/taggd/issues/52#issuecomment-276770955)?

**Comment by sharpner** [comment]
I don't know about the module field, it's the first time that I saw that warning ;) I'll look into it

**Comment by timseverien** [comment]
As noted in #52 ([277801315](https://github.com/timseverien/taggd/issues/52#issuecomment-277801315)), I have made some changed that should improve importing Taggd.

It would be great if you could verify WebPack respects the `module` field.

**Comment by sharpner** [comment]
I did not find a way for webpack 1 yet, (just this issue https://github.com/webpack/webpack/issues/1979) I'll let you know if I found a way... :)

**Comment by timseverien** [comment]
Fixed (see #52)
